### DISCUSSION TITLE
[HELP] Generating Correct TOCTREE for Functions in Documentation

### DISCUSSION THREAD

**Discussion by oualib** [pradyunsg/furo#743] [Questions] [answered]
I hope this message finds you well. I am currently working on documenting a project using Sphinx Furo, and I am facing an issue with the Table of Contents Tree (TOCTREE) for functions.

In my documentation, I have included several titles and subtitles, but the right-side menu seems to display only "init" for functions. This behavior is not ideal, especially for longer documentation where it is essential to navigate through various sections easily.

<img width="1091" alt="Screenshot 2023-12-23 at 03 53 05" src="https://github.com/pradyunsg/furo/assets/32390048/f12e06f4-c735-4834-99ec-8a01ea422a9c">
<img width="1062" alt="Screenshot 2023-12-23 at 03 53 17" src="https://github.com/pradyunsg/furo/assets/32390048/37bf6642-e2fc-4590-a306-91a9b3e28f5e">

Is there a way to do it, in my example: titles "Examples", "Load data for Machine Learning" are missing.

**Comment by pradyunsg** [comment, marked as answer]
This is behaviour that comes from Sphinx, and sadly I can't help you with that. Please reach out to the Sphinx maintainers for assistance with this.

https://www.sphinx-doc.org/en/master/support.html
